### DISCUSSION TITLE
Possible to use GetOrganelle with an Arm64/MacOS?

### DISCUSSION THREAD

**Discussion by antomicblitz** [Kinggerm/GetOrganelle#148] [Q&A]
Hi,

I was trying to install GetOrganelle using anaconda but it seems there are some dependencies which are incompatible. Is it possible to use GetOrganelle on an M1 mac? I am a bit new so I apologise that I am not so proficient in figure this out yet.

Thank you!

Code: 
        
```
(base) antoniolamb@Antonios-MBP ~ % conda install getorganelle

Collecting package metadata (current_repodata.json): done

Solving environment: failed with initial frozen solve. Retrying with flexible solve.

Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.

Collecting package metadata (repodata.json): done

Solving environment: failed with initial frozen solve. Retrying with flexible solve.

Solving environment: - 

Found conflicts! Looking for incompatible packages.

This can take several minutes.  Press CTRL-C to abort.

failed            `                                                      

UnsatisfiableError: The following specifications were found to be incompatible with each other:
Output in format: Requested package -> Available versions
```
<img width="567" alt="Screenshot 2022-03-19 at 15 50 39" src="https://user-images.githubusercontent.com/5877164/159126078-1ef3d72a-2d00-4252-89b2-c7ef75c68d3a.png">

**Comment by antomicblitz** [comment]
Hi! I realize no one responded to this message but just want to say that I found a decent workaround in the meantime. enabling Rosetta emulation and then running the terminal to install conda and then install getorganelle seems to work.

**Comment by Kinggerm** [comment]
Hi,

I was traveling last a few days. Sorry for the delay.

I don't have an M1 machine to test. But according to this issue https://github.com/Kinggerm/GetOrganelle/issues/127, the non-conda approach seems to be working now. You may use a [`pip installing`](https://github.com/Kinggerm/GetOrganelle/wiki/Installation#option-2-using-the-setuppy) approach if you find Rosetta emulation unsatisfied.
